ra/1.1/resource-agent-api.md

```diff
@@ -5,118 +5,129 @@ AT https://github.com/ClusterLabs/OCF-spec/ TO DISCUSS CHANGES**
+purposes only.
 
-It does NOT define any libraries or helper functions which RAs might share
-with regard to common functionality like external command execution, cluster
-logging et cetera, as these are NOT specific to RA and are defined in the
-respective standards.
 
+## Copyright Notice
 
-### API version described
+Originally Copyright 2002,2018 Lars Marowsky-Brée
+Later changes copyright 2020-2021 the Open Cluster Framework project contributors
 
-This document currently describes version 1.1 of the API. 
+Permission is granted to copy, distribute and/or modify this document
+under the terms of the GNU Free Documentation License, Version 1.2 or
+any later version published by the Free Software Foundation; with no
+Invariant Sections, no Front-Cover Texts, and no Back-Cover Texts. A
+copy of the license can be found at https://www.gnu.org/licenses/fdl.txt.
 
 
 ## Terms used in this document
 
-### "Resource"
+### Resource
+
+A _resource_, also known as a _resource instance_, is a logical entity that
+provides a particular computer-hosted service. Examples of resources include a
+disk volume, a network address, a web server, or a virtual machine.
+
+### Cluster
+
+A _cluster_ is a collection of one or more computers under common
+administration running a set of resources.
+
+### Resource Manager
 
-A single physical or logical entity that provides a service to clients or
-other resources. For example, a resource can be a single disk volume, a
-particular network address, or an application such as a web server. A resource
-is generally available for use over time on two or more nodes in a cluster,
-although it usually can be allocated to only one node at any given time.
+A _resource manager_ (RM), also known as a _cluster resource manager_ (CRM),
+is software that manages resources in a cluster.
 
-Resources are identified by a name that must be unique to the particular
-resource type. This is any name chosen by the administrator to identify
-the resource instance and passed to the RA as a special environment
-variable.
+### Resource Type
 
-A resource may also have instance parameters which provide additional
-information required for Resource Agent to control the resource.
+A _resource type_ is a name indicating the service provided by a resource. This
+name should be suitable for use as a file name.
 
+A resource type beginning with a leading dot (.) is a hint to RMs and other
+tools that the resource type should be omitted from lists provided in response
+to user queries.
 
-### "Resource types"
+### Resource Agent
 
-A resource type represents a set of resources which share a common set of
-instance parameters and a common set of actions which can be performed on
-resource of the given type.
+A _resource agent_ (RA) is a software application implementing the RA API for a
+particular resource type. An RA allows a resource manager to perform specific
+mangement tasks for resource instances.
```

Comment:
s/mangement/management/

Reply:
Updated "corrections" commit with this fix (and carried it over to 1.1 in updated "bring in proposed changes" commit)